ovs-p4rt/ovs_p4rt_session.cc

```diff
@@ -84,12 +84,29 @@ absl::StatusOr<std::unique_ptr<OvsP4rtSession>> OvsP4rtSession::Create(
 // destructed.
 absl::StatusOr<std::unique_ptr<OvsP4rtSession>> OvsP4rtSession::Create(
     const std::string& address,
-    const std::shared_ptr<grpc::ChannelCredentials>& credentials,
     uint32_t device_id, absl::uint128 election_id) {
-  return Create(CreateP4RuntimeStub(address, credentials), device_id,
+  return Create(CreateP4RuntimeStub(address, getChannelCredentials()), device_id,
                 election_id);
 }
 
+std::shared_ptr<::grpc::ChannelCredentials> OvsP4rtSession::getChannelCredentials() {
+  std::string cert, key, ca;
+
+  // TODO(5abeel): Hardcoding default certificate file locations for now.
+  readFile("/usr/share/stratum/certs/ca.crt", ca);
+  readFile("/usr/share/stratum/certs/client.key", key);
+  readFile("/usr/share/stratum/certs/client.crt", cert);
+  
+  grpc::SslCredentialsOptions opts = { ca ,key ,cert };
```

Comment:
The space should go _after_ the comma, not _before_ it, just as it does in normal English writing. This spacing is bizarre and jarring. It detracts from the readability of the code.